### DISCUSSION TITLE
Problem with Send to Kindle (Google Play book)

### DISCUSSION THREAD

**Discussion by swissunicorn** [noDRM/DeDRM_tools#508] [Q&A]
I bought an ebook from the Google Play Store and followed all the instructions for how to use the tool. My Calibre is 5.37, I have the latest version of DeDRM (downloaded today), configured the add-on with my Kindle serial number, I downloaded the Adobe Digital Editions app and created an Adobe ID from the email with which I purchased the ebook... Put the ebook in Adobe Digital Editions, restarted Calibre, and then uploaded the .epub file found in Documents > My Digital Editions (conversion went fine). I also got an add-on for KFX input, even though I suspect that's not relevant here.

I have been using Send to Kindle for years with no issue, so I don't think the problem is that. But when I attempt to send my ebook to Kindle, I get an email error: "There was a problem with the document you sent to Kindle" stating that it could not be delivered (either because of the file type not being supported or the document being passport protected / encrypted). I can only guess that the DRM was not properly removed, as the format of the file is EPUB, which is supported.

I would really appreciate any help. 

update: v10.0.9 also does not fix the problem

**Comment by ElleKayEm** [comment]
Can you open the book with calibre's ebook viewer?

**Reply by swissunicorn** [reply]
Yes

**Reply by ElleKayEm** [reply]
Then the issue is not DRM.
